config/nginx.conf

```
@@ -65,5 +65,15 @@ http {
             log_not_found off;
             deny all;
         }
+
+        # allow fpm ping and status from localhost
+        #
+        location ~ ^/(fpm-status|fpm-ping)$ {
+            access_log off;
+            allow 127.0.0.1;
+            deny all;
+            include fastcgi_params;
```

PHP-FPM currently doesn't know what script you're requesting. So it now returns a 200, but it doesn't give any response body.
You can fix that by adding the `SCRIPT_FILENAME` like this;
```suggestion
            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
            include fastcgi_params;
```